designcompose/src/main/java/com/android/designcompose/proto/DimensionProto.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose.proto
+
+import com.android.designcompose.serdegen.Dimension
+import com.android.designcompose.serdegen.DimensionProto
+import java.util.*
+
+fun DimensionProto.get(): Dimension = dimension.orElseThrow { NoSuchFieldException("Malformed data: DimensionProto's dimension unset") }
```

Comment:
What is `dimension`? I looked at the generated `DimensionProto` file but didn't see that as a member variable.